image-customization.lua

```diff
@@ -22,6 +22,7 @@ features({
 packages({
 	'iwinfo',
 	'respondd-module-airtime',
+	'ffda-ssh-manager',
```

Comment:
The package actually seems to be called `ffda-gluon-ssh-manager`?

https://github.com/freifunk-gluon/community-packages/tree/master/ffda-gluon-ssh-manager

Reply:
No. https://github.com/freifunk-gluon/community-packages/blob/master/ffda-gluon-ssh-manager/Makefile#L21

Reply:
okay, wasn't aware that dir and package name can be different. Would've expected that it's part of the Search path.
I would prefer if they were identical (less confusion), but i guess it's okay if it works.